bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/widgets/Table.java

```diff
@@ -3477,6 +3477,9 @@ void tableViewSelectionDidChange (long id, long sel, long aNotification) {
 
 @Override
 void tableViewSelectionIsChanging (long id, long sel, long aNotification) {
+	if (keyDown) {
```

Comment:
Can we make it `if (keyDown) return;` to keep the style consistent across?

Reply:
Also, will be useful to add a comment here on why this check is required.

Reply:
sure, I will do that. Should I add a link to the developer.apple site or is this forbidden?

Reply:
A short description of why the code is requried would help. Links to specific doc in apple developer site change sometimes or the content changes then the link may not be useful.